wire-java-generator/src/main/java/com/squareup/wire/java/JavaGenerator.java

```diff
@@ -218,6 +236,12 @@ public TypeName typeName(ProtoType protoType) {
     return candidate;
   }
 
+  public TypeName wireTypeName(ProtoType protoType) {
```

Comment:
not sure how I feel about this. Do we really need it? I’d prefer if `typeName` just worked and the caller didn’t need to decide.